keepassxc-browser/options/options.html

```diff
@@ -130,7 +130,12 @@ <h2>General Settings</h2>
             <label class="checkbox">
               <input type="checkbox" name="showNotifications" value="true" /> Show notifications.
             </label>
-            <span class="help-block">Allow KeePassXC-Browser to show notifications when user interaction is needed from the extension icon.</span>
+            <span class="help-block">Allow KeePassXC-Browser to show notifications about alerts and when user interaction is needed from the extension icon.</span>
+          </div>
+          <div class="checkbox">
+            <label class="checkbox">
+              <input type="checkbox" name="showLoginNotifications" value="true" /> Show notification also when new credentials are detected.
```

Comment:
Same here, the wording does not convey the intent.

Reply:
Maybe this could be better: "Show notifications when unsaved credentials are detected."?

Reply:
How about "Show a notification when new credentials can be saved to the database"

Reply:
Done.

Reply:
Thanks